testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -440,9 +483,18 @@ public void shouldUsePaginationForSimpleStatement()
             Thread.sleep(100);
```

Comment:
maybe we could import the Awaitality library for test blocking code?
https://github.com/awaitility/awaitility
Then it could be changed to 
`await().atMost(100, MILLISECONDS).until(actionToWait());`

Reply:
ok, I've filed a ticket for it.

https://datastax.jira.com/browse/C2-307

Reply:
nice, thx